sale_multi_payment_confirm/tests/test_sale_multi_payment_confirm.py

```diff
@@ -0,0 +1,20 @@
+from odoo import tests
+
+from odoo.addons.sale.tests.test_sale_transaction import TestSaleTransaction
+
+
+@tests.tagged("post_install", "-at_install")
+class TestSaleTransaction(TestSaleTransaction):
+    def test_00_sale_multi_payment_confirm(self):
+        self.transaction.amount -= 10
```

Comment:
what's the need to set vars on `self`?

Reply:
`self.transaction` is already assigned by the parent class from which we inherit: https://github.com/odoo/odoo/blob/14.0/addons/sale/tests/test_sale_transaction.py#L28

I only added `self.transaction2` for consistency.